### PR TITLE
Add specs for all German, Dutch and English deviation fragments

### LINKED ISSUE #520: Improve syllable count for Dutch
We should improve the syllable count for Dutch for the following cases:
- linedance: now counted as 3 (li-ne-dance)

### LINKED ISSUE #848: Make sure there is a spec for every subtractSyllable and addSyllable config
In #837 we introduced a ton of configuration for the dutch Flesch reading ease score. We should make sure that all `subtractSyllable` and `addSyllable` config is covered in a spec to make sure one cannot be deleted without having at least one example that needs that configuration.

### PR DESCRIPTION
**Pull request by IreneStr**
Adds specs for all deviation fragments (add syllables and subtract syllables) for English, German and Dutch. Also removes redundant and incorrect deviation fragments. Also adds 'linedance' to Dutch global fragments.

This PR can be summarized in the following changelog entry: Improved syllable count for the English, Dutch and German Flesch Reading Ease.

Fixes #848 
Fixes #520

### CODE REVIEW

File: spec/stringProcessing/syllables/countSpec.js [hunk 1]
```diff
@@ -163,18 +768,567 @@ describe( "a syllable counter for Dutch textstrings", function () {
...
+
+	it( "returns the number of syllables of words containing the add syllable ä[aeoi]", function () {
+		expect( countSyllableFunction( "mäandrisch", "de_DE" ) ).toBe( 3 );
+		expect( countSyllableFunction( "achäer", "de_DE" ) ).toBe( 3 );
+		expect( countSyllableFunction( "äolisch", "de_DE" ) ).toBe( 3 );
+		expect( countSyllableFunction( "achäisch", "de_DE" ) ).toBe( 3 );
+	} );
+
+	it( "returns the number of syllables of words containing the add syllable auä", function () {
+		expect( countSyllableFunction( "blauäugig", "de_DE" ) ).toBe( 3 );
```
Comment (terw-dan) [inline spec/stringProcessing/syllables/countSpec.js]: This isn't covered by the fragment `uäu` but by `auä`. So either the `uäu` is redundant, or it should be tested with another word.

File: spec/stringProcessing/syllables/countSpec.js [hunk 2]
```diff
@@ -163,18 +768,567 @@ describe( "a syllable counter for Dutch textstrings", function () {
...
+
+	it( "returns the number of syllables of words containing the subtract syllable guette", function () {
+		expect( countSyllableFunction( "baguette", "de_DE" ) ).toBe( 2 );
+	} );
+
+	it( "returns the number of syllables of words containing the subtract syllable ête$", function () {
+		expect( countSyllableFunction( "tête", "de_DE" ) ).toBe( 1 );
+	} );
+
+	it( "returns the number of syllables of words containing the subtract syllable ôte$", function () {
```
Comment (terw-dan) [inline spec/stringProcessing/syllables/countSpec.js]: If `ôte$` is removed, the test still passes.

File: js/config/syllables/de.json [hunk 3]
```diff
@@ -1,455 +1,461 @@
...
+					{ "word": "hornpipe", "syllables": 2, "notFollowedBy": ["n", "r"] },
+					{ "word": "hotline", "syllables": 2, "notFollowedBy": ["n", "r"] },
+					{ "word": "infoline", "syllables": 3, "notFollowedBy": ["n", "r"] },
+					{ "word": "inline", "syllables": 2, "notFollowedBy": ["n", "r"] },
+					{ "word": "kite", "syllables": 1, "notFollowedBy": ["n", "r"] },
+					{ "word": "rollerblade", "syllables": 1, "notFollowedBy": ["n", "r"] },
+					{ "word": "score", "syllables": 1, "notFollowedBy": ["n", "r"] },
+					{ "word": "skyline", "syllables": 2, "notFollowedBy": ["n", "r"] },
+					{ "word": "slackline", "syllables": 2, "notFollowedBy": ["n", "r"] },
+					{ "word": "slice", "syllables": 1, "notFollowedBy": ["n", "r"] },
```
Comment (terw-dan) [inline js/config/syllables/de.json]: Shouldn't this not be followed by an N R and S (slices has 2 syllables).

**Comment by terw-dan** [comment]
CR done.

File: spec/stringProcessing/syllables/countSpec.js (hunk 1 above)
Comment (IreneStr) [inline spec/stringProcessing/syllables/countSpec.js]: stauänderung, zuäußerst

File: spec/stringProcessing/syllables/countSpec.js (hunk 2 above)
Comment (IreneStr) [inline spec/stringProcessing/syllables/countSpec.js]: One test fails when ôte$ is removed.

**Comment by terw-dan** [comment]
Acceptance :+1:

**Pull request merged**